src/components/AppButton/AppButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement, PropsWithChildren } from 'react';
+
+type AppButtonVariant = 'appButton' | 'appButton_thin';
```

Comment:
excellent usage of type. One minor: stick to full camelCase notation: appButtonThin